src/cpp.rs

```diff
@@ -1654,7 +1654,7 @@ pub mod root {
                     #[link_name = "\u{1}_ZN3app8lua_bind31ControlModule__get_trigger_implEPNS_26BattleObjectModuleAccessorE"]
                     pub fn get_trigger(
```

Comment:
For example, this function would be nice to have documentation for. I don't really know what it's returning.

Reply:
Added a bunch of rustdocs to some functions that I personally used a bunch. Lemme know if they look alright. For the ones that don't have docs, I generally don't know what they do specifically and was just generally trying to fill in return types to ones I could very reasonably guess. If you'd prefer I test all of them, I'm happy to do so and document them when I am able.